terraform/eks_fargate_setup/oidc_thumbprint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ -z "${ATLAS_WORKSPACE_SLUG}" ]]; then
+  APP="tail -r"
+else
+  APP="tac"
+fi
+
+THUMBPRINT=$(echo | openssl s_client -servername oidc.eks.${1}.amazonaws.com -showcerts -connect oidc.eks.${1}.amazonaws.com:443 2>&- | ${APP} | sed -n '/-----END CERTIFICATE-----/,/-----BEGIN CERTIFICATE-----/p; /-----BEGIN CERTIFICATE-----/q' | ${APP} | openssl x509 -fingerprint -noout | sed 's/://g' | awk -F= '{print tolower($2)}')
```

Comment:
Has this been tested? Maybe add a comment describing the steps this takes.

Reply:
Yes I have tested this. 
THUMBPRINT=$(echo | openssl s_client -servername oidc.eks.us-west-2.amazonaws.com -showcerts -connect oidc.eks.us-west-2.amazonaws.com:443 2>&- | tail -r | sed -n '/-----END CERTIFICATE-----/,/-----BEGIN CERTIFICATE-----/p; /-----BEGIN CERTIFICATE-----/q' | tail -r | openssl x509 -fingerprint -noout | sed 's/://g' | awk -F= '{print tolower($2)}')

Reply:
{"thumbprint": -thumb print is here-}